docs/proposals/volume-provisioning.md

```diff
@@ -485,15 +485,55 @@ parameters:
    type: spinning
 ```
 
+### AWS provisioner and a PVC w
+
+This example shows how a PVC can require a volume from given zone. If there is
+a matching available PV, it will be bound to the PVC. If no such PV exists,
+a new one in the right zone will be provisioned by AWS provisioner.
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: myclaim
+  annotations:
+    "volume.beta.kubernetes.io/storage-class": "aws-fast"
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1500Mi
+  selector:
+    matchLabels:
+      "failure-domain.beta.kubernetes.io/zone": "us-east-1a"
```

Comment:
On unrelated why are we calling the label key `failure-domain.beta.kubernetes.io/zone` ? Can we not keep it something simple like `zone`?  I understand that - pvc will get `failure-domain.beta.kubernetes.io/zone` label after it gets provisioned but I always understood `failure-domain.beta.kubernetes.io/zone` as a internal detail.